modules/social_features/social_profile/modules/social_profile_fields/social_profile_fields.install

```diff
@@ -141,3 +156,10 @@ function social_profile_fields_update_121001(): void {
     }
   }
 }
+
+/**
+ * Add new permissions for taxonomies.
```

Comment:
@Macsym2018 Shouldn't we update other permissions here?

```
$permissions = [
    'view terms in nationality',
    'view term names in nationality',
    'select terms in nationality',
  ];
  user_role_grant_permissions('sitemanager', $permissions);
```

Reply:
@ribel These permissions are granted in social_profile_fields_update_121001

Reply:
@Macsym2018 And what about create/edit/delete terms permissions?
We actually need to make sure they are set for SM.

Reply:
@ribel Added